risc0/zkvm/Cargo.toml

```diff
@@ -68,10 +68,11 @@ tracing = { version = "0.1", default-features = false, features = [
 
 # Host dependencies
 [target.'cfg(not(target_os = "zkvm"))'.dependencies]
-addr2line = { version = "0.24.2", optional = true, features = [
+addr2line = { version = "0.24.2", features = [
   "rustc-demangle",
   "std",
-] }
+], optional = true }
```

Comment:
Could you revert this part? We're trying to have the `features = ` be the last part of the dependency line since it's almost always dynamic.